docusaurus.config.js

```diff
@@ -55,13 +55,15 @@ module.exports = {
           editUrl:
             'https://github.com/kwilteam/',
         },
+        /* no blog here yet, only a custom link
```

Comment:
`Warn: `blogDir` doesn't exist: "/home/jon/kwil/git/docs/blog"`
afaict, docusaurus is right and the blog was done externally.  We only make a manual link to it on the Links page.